homeassistant/components/microsoft_face/__init__.py

```diff
@@ -27,7 +27,7 @@
 DEFAULT_TIMEOUT = 10
 DOMAIN = "microsoft_face"
 
-FACE_API_URL = "api.cognitive.microsoft.com/face/v1.0/{0}"
+FACE_API_URL = "cognitiveservices.azure.com/face/v1.0/{0}"
```

Comment:
I suggest this be moved to the configuration schema and default to this URL.  That way, if someone wants to use the trial URL, they can configure it temporarily.

Reply:
why would trials have a different url 😢 I prefer not to put the burden of finding this url on users if we can.

Reply:
Sure, the config could be “is_trial” and the logic to use the appropriate url is in the code.